config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml

```diff
@@ -8,7 +8,7 @@ periodics:
   extra_refs:
   - org: jetstack
     repo: cert-manager
-    base_ref: release-1.1
+    base_ref: release-1.2
```

Comment:
Nightly tests to now run against the 1.2 release.
 * https://testgrid.k8s.io/jetstack-cert-manager-previous